checkpoint_ngfw/plugin.spec.yaml

```diff
@@ -692,6 +692,50 @@ actions:
         type: boolean
         required: true
 
+  set_threat_protection:
+    title: Set Threat Protection
+    description: Set threat protection
```

Comment:
Anythign we can put here that describe what threat protection does? Same in help.md